sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryOptionsExtensions.kt

```diff
@@ -28,11 +30,29 @@ internal fun CocoaSentryOptions.applyCocoaBaseOptions(options: SentryOptions) {
     this.enableAutoSessionTracking = options.enableAutoSessionTracking
     this.beforeSend = { event ->
         dropKotlinCrashEvent(event as NSExceptionSentryEvent?) as SentryEvent?
+
+        val cocoaName = BuildKonfig.SENTRY_COCOA_PACKAGE_NAME
+        val cocoaVersion = BuildKonfig.SENTRY_COCOA_VERSION
+        options.sdk.apply {
+            val names = this.packages?.map { it.name }
+            if (names?.contains(cocoaName) != true) {
+                this.addPackage(cocoaName, cocoaVersion)
+            }
+        }
+        val sdk = event?.sdk?.toMutableMap()
+        sdk?.set(
+            "packages",
+            options.sdk.packages?.map {
+                mapOf("name" to it.name, "version" to it.version)
+            }
+        )
+        event?.sdk = sdk
```

Comment:
Can we do that during the SDK init instead using the `PrivateSentrySDKOnly`?
Packages need to be added in beforeSend, AFAIK there's no other way yet https://github.com/getsentry/sentry-dart/blob/11fb40841ad3d6fc46e02eb8adeb74d5b6932582/flutter/ios/Classes/SentryFlutterPluginApple.swift#L233-L250